app/views/publishers/vacancies/job_applications/_banner.html.slim

```diff
@@ -26,7 +26,7 @@
                 = t(".deadline.before")
             span.govuk-body = format_time_to_datetime_at(vacancy.expires_at)
 
-      - unless vacancy.draft?
+      - unless vacancy.draft? || vacancy.expires_at < 1.year.ago
```

Comment:
Please either use the scope `created_within_data_access_period` or the constant `DATA_ACCESS_PERIOD_FOR_PUBLISHERS`

Reply:
Or write a predicate method on vacancy

Reply:
have used the constant